k8s/release/mailrelay/mailrelay/patches/dev/cluster-00/mailrelay.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   values:
     image:
       repository: sdshmctspublic.azurecr.io/exim-relay
-      tag: 0.9
+      tag: 1.0
```

Comment:
Can you make it 0.10 please? It's still work in progress

Reply:
updated accordingly